File: tests/fixtures/typescript/env_refs.ts
// Direct access
const dbUrl = process.env.DATABASE_URL;

// With fallback (nullish coalescing)
const port = process.env.PORT ?? "3000";

// With fallback (logical OR)
const host = process.env.HOST || "localhost";

// No fallback
const apiKey = process.env.API_KEY;

// Destructured
const { NODE_ENV, SECRET_KEY } = process.env;

File: tests/fixtures/python/env_refs.py
import os

# Direct access (no default)
database_url = os.environ["DATABASE_URL"]

# With default via os.environ.get
port = os.environ.get("PORT", "5432")

# With default via os.getenv
host = os.getenv("HOST", "localhost")

# No default via os.getenv
api_key = os.getenv("API_KEY")

# Direct bracket access
secret = os.environ["SECRET_KEY"]
